File: LAB_9_10/LAB10 20161811_20161442/LAB10 20161811_20161442/GameSoftLab10CS/GameSoft/GameSoftRRHH/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameSoftRRHH
{
    public class Location
    {
        private string _address;
        private string _postalCode;
        private string _city;
        private string _province;
        private Country _country;

        public string Address { get => _address; set => _address = value; }
        public string PostalCode { get => _postalCode; set => _postalCode = value; }
        public string City { get => _city; set => _city = value; }
        public string Province { get => _province; set => _province = value; }
        public Country Country { get => _country; set => _country = value; }
    }
}

File: LAB_9_10/LAB10 20161811_20161442/LAB10 20161811_20161442/GameSoft-Netbeans/GameSoftRRHH/src/pucp/gamesoft/rrhh/model/Location.java
package pucp.gamesoft.rrhh.model;

public class Location{
	private String address;
	private String postalCode;
	private String city;
	private String province;
	private Country country;
	
	public Location(String address, String postalCode, String city, String province, Country country){
		this.address = address;
		this.postalCode = postalCode;
		this.city = city;
		this.province = province;
		this.country = country;
	}
	
	public String getAddress(){
		return address;
	}
	
	public void setAddress(String address){
		this.address = address;
	}
	
	public String getPostalCode(){
		return postalCode;
	}
	
	public void setPostalCode(String postalCode){
		this.postalCode = postalCode;
	}
	
	public String getCity(){
		return city;
	}
	
	public void setCity(String city){
		this.city = city;
	}
	
	public String getProvince(){
		return province;
	}
	
	public void setProvince(String province){
		this.province = province;
	}
}